src/components/SimpleInput.js

```diff
@@ -0,0 +1,55 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      type: 'text',
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <input
+        {...this.props}
+        name={this.props.name}
+        style={Object.assign({}, styles.wrapper, this.props.style)}
+        type={this.props.type}
```

Comment:
@cerinman we should remove `style` because this component won't handle different types such as `checkbox`, `radio`, etc passed through `this.props.style`. 

@jmophoto are you suggesting we add `onChange` to the `props`?